strapi/strapi
# Add primary keys to relational table

### What does it do?

Add Primary Keys to link tables

### Why is it needed?

Some cloud service require them

### How to test it?

- TO BE Tested on multiple DBs
- Start with all code and create data (relations)
- Stop server
- Switch to this PR
- Restart
- No errors should happen and Ids should be created in join tables 


### Related issue(s)/PR(s)

- fix https://github.com/strapi/strapi/issues/13455

### TODO

- [x] It breaks migrations on existing Sqlite databases we still need a solution
